sources/dynamodb/data.go

```diff
@@ -149,22 +197,6 @@ func cvtColValue(attrVal *dynamodb.AttributeValue, srcType string, spType string
 				return nil, fmt.Errorf("failed to convert %v to a json string", attrVal.GoString())
 			}
 			return string(b), nil
-		case typeString:
-			return *attrVal.S, nil
-		case typeStringSet:
```

Comment:
Do we retain the old functionality for non-pg spanner post these changes ?

Reply:
Yes we do. I made the changes accordingly to support both and integration tests for dydb passing means older version works fine.